src/include/cpu/rpp_cpu_simd.hpp

```diff
@@ -2438,6 +2438,29 @@ static inline __m128 log_ps(__m128 x)
     return x;
 }
 
+inline Rpp32f rpp_hsum_ps(__m128 x)
+{
```

Comment:
@r-abishek Are we not following Google style coding for RPP? With all opening { on the same line as function name

Reply:
Once the BatchPD->Tensor migration is complete and we are in a position to phase out BatchPD code, there will be a lot less code to deal with, and a single PR to format the whole repo could be done to adhere to google std.
For now, there's easily a 90+% majority of code in RPP in this format, so retaining this